src/css/application.css

```diff
@@ -582,6 +582,11 @@ body {
   color: var(--color-dark-gray);
 }
 
+.pop-throbber__image--thinking {
+  background: url('../images/pop/thinking.svg') center no-repeat;
```

Comment:
Not that crazy? Especially if it were done cleanly, like if it was required as a template via ejs or something. I'd save it for a future PR though.